### ISSUE TITLE
Finish splitting API from main stac-spec

### ISSUE THREAD

**Issue by matthewhanson** [radiantearth/stac-api-spec#1]
This repo contains the contents of the api-spec folder from stac-spec (0.9.0), but there are numerous links and references that need to be updated.

Additional text should be added that this is the API spec and point to stac-spec.

One question is how do we reference individual pages in the main stac-spec?  Do we reference master from master branch and dev from dev (really annoying), or tag a specific version?

**Comment by cholmes** [comment]
Didn't see this before I started linking. But I linked to master branch. I think it'd be ok to link from dev to master - the idea is that these specs now move semi-independently and cut releases at different times. 

Thinking about it that could get annoying if all 'released' versions in stac-api-spec are referring to master, as things change. I'd say the ideal would be that a released version of stac-api-spec should depend on a specific version of stac-spec, and ideally we'd update the links to point at those. That sounds like it could be annoying, though maybe that should just spur us to limit the number of links.
